jobsdb/jobsdb.go

```diff
@@ -1521,8 +1520,7 @@ func (jd *HandleT) createDS(newDS dataSetT, l lock.DSListLockToken) {
 	// In case of a migration, we don't yet update the in-memory list till we finish the migration
 	if l != nil {
 		// to get the updated DS list in the cache after createDS transaction has been committed.
-		_ = jd.refreshDSList(l)
```

Comment:
Can you explain why this line was removed?

Reply:
I can, since I requested it :) 
It is redundant, the next line calls another method which internally performs a refresh too.

Reply:
Makes sense, thanks :+1: